vision_msgs_rviz_plugins/README.md

```diff
@@ -0,0 +1,45 @@
+# vision_msgs_rviz_plugins
+
+This package contains a RVIZ2 plugin to display (some) vision_msgs for ROS 2.
```

Comment:
Some? This looks like all :wink

Reply:
Was thinking of adding BoundingBox2D/Array in the future when I need it for my project